src/collection.js

```diff
@@ -440,6 +440,16 @@ class Collection {
     return this.items.slice(0)
   }
 
+  /**
+  * Returns reversed version of original collection
+  *
+  * @returns {Array}
+  */
+  reverse () {
+    this.items = this.items.reverse()

```

Comment:
Do you need the assignment here? Does `Array#reverse()` return the array in reversed order or does it change the original array?

Reply:
Probably I don't. Because `Array#reverse()` either reverses the original array and returns a reference to it, so the references on the left side and on the right side are the same. I will return the assignment here. Thank you for the remark!